tests/functional/prometheous/test_prometheus_validations.py

```diff
@@ -974,3 +975,82 @@ def test_restart_prometheus_glusterfs_pod(self):
             # Try to fetch metric from prometheus pod
             self._fetch_metric_from_promtheus_pod(
                 metric='heketi_device_brick_count')
+
+    @pytest.mark.tier2
+    def test_metrics_workload_on_prometheus(self):
+        """Validate run metrics workload on prometheus"""
+
+        self.prefix = "autotest-{}".format(utils.get_random_str())
+
+        # Add check for CRS version
+        openshift_ops.switch_oc_project(
+            self._master, self._registry_project_name)
+        if not self.is_containerized_gluster():
+            self.skipTest("Skipping this test case as CRS"
+                          " version check can not be implemented")
```

Comment:
Test case doesn't suggest executing only on converged mode setup. This check is not required.